src/rez/wheel.py

```diff
@@ -0,0 +1,570 @@
+
+    https://www.python.org/dev/peps/pep-0427/#file-contents
+
+    Arguments:
+        wheel (str): Contents of a WHEEL file
+
+    Returns:
+        variants (dict): With keys {"platform", "os", "python"}
+
+    """
+
+    variants = {
+        "platform": None,
+        "os": None,
+        "python": None,
+    }
+
+    py = {
+        "2": False,
+        "3": False,
+        "minor": False,
+    }
+
+    for line in wheel.splitlines():
+        line = line.rstrip()
+
+        if not line:
+            # Empty lines are allowed
+            continue
+
+        line = line.replace(" ", "")
+        key, value = line.lower().split(":")
+
+        if key == "wheel-version":
+            if value[0] != "1":
+                raise ValueError("Unsupported WHEEL format")
+
+        if key == "root-is-purelib" and value == "false":
+            variants["platform"] = platform_name()
+
+        if key == "tag":
+            # May occur multiple times
+            #
+            # Example:
+            #   py2-none-any
+            #   py3-none-any
+            #   cp36-cp36m-win_amd64
+            #
+            py_tag, abi_tag, plat_tag = value.split("-")
+            major_ver = py_tag[2]
+
+            py[major_ver] = True
+
+            if plat_tag != "any":
+                # We could convert e.g. `win_amd64` to a Rez platform
+                # and os version, such as `platform-windows` and
+                # `os-windows.10.0.1800` but it's safe to assume that if
+                # this package was provided by pip, it must be specific
+                # to the currently running platform and os.
+
+                variants["os"] = os_name()
+                variants["platform"] = platform_name()  # e.g. windows
+
+                # Indicate that this week depends on the Python minor version
+                # which is true of any compiled Python package.
+                py["minor"] = True
+
+    if py["minor"]:
+        # Use the actual version from the running Python
+        # rather than what's coming out of the the WHEEL
+        # See 
+        variants["python"] = python_version()
+
+    elif py["2"] and py["3"]:
+        variants["python"] = None
+
+    elif py["2"]:
+        variants["python"] = "2"
+
+    elif py["3"]:
+        variants["python"] = "3"
+
+    return [
+        k + "-" + variants[k]
+
+        # Order is important
+        for k in ("platform",
+                  "os",
+                  "python")
+
+        if variants[k] is not None
+    ]
+
+
+def os_name():
+    """Return pip-compatible OS, e.g. windows-10.0 and Debian-7.6"""
+    # pip packages are no more specific than minor/major of an os
+    # E.g. windows-10.0.18362 -> windows-10.0
+    return ".".join(platform_.os.split(".")[:2])
+
+
+def platform_name():
+    return platform_.name
+
+
+@lru_cache()
+def python_version():
+    """Return major.minor version of Python, prefer current context"""
+
+    import subprocess
+    from rez.status import status
```

Comment:
subprocess is already imported at the top of the file. Also, could `rez.status` be moved at the top of the file too?